test/unit/testIhtPaper.js

```diff
@@ -14,32 +14,45 @@ describe('IhtPaper', () => {
         });
     });
 
-    describe('handlePost()', () => {
-        let ctx;
-        let errors;
-        let formdata;
-        let session;
-        let hostname;
-        let featureToggles;
-
-        it('should return the ctx with the deceased married status and the screening_question feature toggle', (done) => {
-            ctx = {
+    describe('getContextData()', () => {
+        it('should return the ctx with the deceased address and the screening_question feature toggle on', (done) => {
+            const req = {
+                sessionID: 'dummy_sessionId',
+                session: {form: {}, featureToggles: {screening_questions: true}},
+                body: {
+                    form: 'IHT205',
+                    grossIHT205: '500000',
+                    netIHT205: '400000'
+                }
+            };
+            const ctx = IhtPaper.getContextData(req);
+            expect(ctx).to.deep.equal({
                 form: 'IHT205',
                 grossIHT205: '500000',
-                netIHT205: '400000'
+                netIHT205: '400000',
+                isToggleEnabled: true,
+                sessionID: 'dummy_sessionId'
+            });
+            done();
+        });
+
+        it('should return the ctx with the deceased address and the screening_question feature toggle off', (done) => {
```

Comment:
Please update the description for this test as it is for iht paper but it is referring to deceased address.